aloe_django/steps/models.py

```diff
@@ -54,33 +58,43 @@ def _models_generator():
 _WRITE_MODEL = {}
 
 
-def creates_models(model):
-    """
-    Register a model-specific creation function. Wrapper around writes_models
-    that removes the field parameter (always a create operation).
+def writes_models(model):
     """
+    Register a model-specific create and update function.
 
-    def decorated(func):
+    This can then be accessed via the steps.
 
-        @wraps(func)
-        @writes_models(model)
-        def wrapped(data, field):
-            if field:
-                raise NotImplementedError(
-                    "Must use the writes_models decorator to update models")
-            return func(data)
+        And I have foos in the database:
+            | name | bar  |
+            | Baz  | Quux |
 
-    return decorated
+        And I update existing foos by pk in the database:
+            | pk | name |
+            | 1  | Bar  |
 
+    A method for a specific model can define a function write_badgers(data,
+    field), which creates and updates the Badger model and decorating it with
+    the writes_models(model_class) decorator.
 
-def writes_models(model):
-    """
-    Register a model-specific create and update function.
+    @writes_models(Profile)
+    def write_profile(data, field):
+        '''Creates a Profile model'''
+
+        for hash_ in data:
+            if field:
+                profile = Profile.objects.get(**{field: hash_[field]})
+                else:
+                    profile = Profile()
+
+                ...
 
     The function must accept a list of data hashes and a field name. If field
     is not None, it is the field that must be used to get the existing objects
     out of the database to update them; otherwise, new objects must be created
     for each data hash.
+
+    If you only want to modify the hash or do specific changes, you can also
+    modify the hash and then pass it on to write_models().
```

Comment:
You don't talk about the tautology club, because you don't talk about the tautology club.